stratum/hal/lib/dummy/dummy_phal.h

```diff
@@ -62,6 +62,16 @@ class DummyPhal : public PhalInterface {
   SHARED_LOCKS_REQUIRED(chassis_lock)
   LOCKS_EXCLUDED(phal_lock_) override;
 
+  ::util::Status GetOpticalTransceiverInfo(uint64 node_id, uint32 port_id,
+                                           OpticalChannelInfo* oc_info)
+  SHARED_LOCKS_REQUIRED(chassis_lock)
+  LOCKS_EXCLUDED(phal_lock_) override;
+
+  ::util::Status SetOpticalTransceiverInfo(uint64 node_id, uint32 port_id,
+                                           const OpticalChannelInfo& oc_info)
+  SHARED_LOCKS_REQUIRED(chassis_lock)
+  LOCKS_EXCLUDED(phal_lock_) override;
+
```

Comment:
Formatting. I see that the whole file is out of shape, but let's not add more to it.